src/v8runtime/V8Runtime.cpp

```diff
@@ -66,9 +67,10 @@ V8Runtime::V8Runtime(std::unique_ptr<V8RuntimeConfig> config)
   v8::HandleScope scopedHandle(isolate_);
   context_.Reset(isolate_, CreateGlobalContext(isolate_));
   v8::Context::Scope scopedContext(context_.Get(isolate_));
+  this->jsQueue = jsQueue;
```

Comment:
```suggestion
  jsQueue_ = jsQueue;
```
if we use underscore convention, `this->` is unnecessary.